prov/verbs/src/fi_verbs.h

```diff
@@ -104,10 +104,19 @@
 
 #define VERBS_DEF_CQ_SIZE 1024
 #define VERBS_MR_IOV_LIMIT 1
+#define VERBS_DEFAULT_INLINE_SIZE 64
+
+#define STR(X) #X
+#define STR_INT(X) STR(X)
```

Comment:
nit: because these are defines they pollute the entire compile namespace.  Neither macro seems necessary.